azurerm/internal/services/databricks/databricks_customer_managed_key_resource_test.go

```diff
@@ -151,7 +151,7 @@ resource "azurerm_databricks_workspace" "test" {
 }
 
 %[4]s
-`, data.RandomInteger, data.Locations.Primary, keyVault, cmk)
+`, data.RandomInteger, "eastus2", keyVault, cmk)
```

Comment:
could we use the alt location? it is by default east2 in our ci

Reply:
fixed